<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FittingWizzard</class>
 <widget class="QWizard" name="FittingWizzard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit wizzard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWizardPage" name="orientationwizardPage">
   <property name="title">
    <string>Orientation</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="localSensorRadioButton">
      <property name="text">
       <string>Local stick sensor orientation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">orientationButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sensorFrameRadioButton">
      <property name="text">
       <string>Global sensor reference frame orientation</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">orientationButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Cardan angles</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Roll</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="rollDoubleSpinBox">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Pitch</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="pitchDoubleSpinBox">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Yaw</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="yawDoubleSpinBox">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Quaternion</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="yDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>w</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="wDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="offsetWizardPage">
   <property name="title">
    <string>Offset</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="customOffsetRadioButton">
          <property name="text">
           <string>Custom</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">offsetButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="offsetSpinBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="autoOffsetRadioButton">
          <property name="text">
           <string>Auto detect</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">offsetButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="autoOffsetConfigurationGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Step:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="offsetStepSpinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Minimal fit size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="minFitSizeSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>165</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>orientationButtonGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>orientationGroupChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetButtonGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>offsetGroupChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitchDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>cardanAngleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yawDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>cardanAngleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>cardanAngleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>offsetChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minFitSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>minFitSizeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetStepSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FittingWizzard</receiver>
   <slot>offsetStepChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>orientationGroupChanged(int)</slot>
  <slot>offsetGroupChanged(int)</slot>
  <slot>cardanAngleChanged()</slot>
  <slot>quaternionChanged()</slot>
  <slot>offsetChanged(int)</slot>
  <slot>offsetStepChanged(int)</slot>
  <slot>minFitSizeChanged(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="offsetButtonGroup"/>
  <buttongroup name="orientationButtonGroup"/>
 </buttongroups>
</ui>
